kong/clustering/wrpc_control_plane.lua

```diff
@@ -195,30 +208,17 @@ function _M:handle_cp_websocket()
     dp_version = dp_version,
     log_suffix = log_suffix,
     basic_info = nil,
-    basic_info_semaphore = semaphore.new()
+    basic_info_semaphore = semaphore.new(),
+    dp_plugins_map = empty_table,
+    cp_ref = self,
+    config_hash = string.rep("0", 32), -- initial hash
+    sync_status = CLUSTERING_SYNC_STATUS.UNKNOWN,
```

Comment:
We move the initializations here so they will be accessible from `ReportMetadata`.